### DISCUSSION TITLE
I have a poltergeist network problem with LXC Containers (Installed by community scripts) after Proxmox server restart

### DISCUSSION THREAD

**Discussion by flnf** [community-scripts/ProxmoxVE#1087] [Q&A]
I have a Proxmox server where I've been installing LXC containers using the helper scripts (https://community-scripts.github.io/). The Proxmox server has a DNS Name of home.lan and a DNS Server with the IP 192.168.1.240, which corresponds to an LXC container running Pi-hole. Using the helper scripts, I have installed various containers (always assigning a static IP instead of using DHCP), such as Pi-hole with Unbound, Homepage, Debian 12, Uptime Kuma, and Nginx Proxy Manager.

After installation, all containers worked without any issues. However, after rebooting Proxmox, the Homepage and Uptime Kuma containers started experiencing network problems. Strangely, they can ping the IPs of this server, but when I try to ping 192.168.1.1 (router), other IPs (I have another Proxmox server not in a cluster), or google.com, they don’t respond.

I have reviewed the configuration of the containers and their network settings but can’t find any differences compared to the containers that are working fine. Could you help me figure out why this is happening with some containers and not others?

**Comment by Mellowlynx** [comment]
I have observed that if the Pi-hole is not started as the first LXC, that the others LXC will get is some sort of conflict and the resolver config will change.
But this should only happen with domain names, not IP's.
So you should look at your network and firewall settings.
